H_w 3/test_positive.rb

```diff
@@ -202,10 +202,53 @@ def test_bug_types_isue
     expected_text = 'Bug'
     actual_text = type_issue[index].text
     assert_equal(expected_text, actual_text)
+  end
+
+  def test_create_or_not_bug_issue
+    register_user
+    add_new_project
 
+    x = rand(1..2)
+
+    if x == 1
+      add_new_issue
+      puts"Your random action isn't create a new bug issue"
+     else puts"Your random action is create a new bug issue"
+    end
+
+    @driver.find_element(:class,'my-page').click
+
+    @wait.until {@driver.find_element(:class,'mypage-box').displayed?}
+
+    classes_els = @driver.find_elements(:css,'.subject > a')
+    my_element = classes_els.find {|el| el.text.include? @bug_issue}
+
+    if my_element == nil
```

Comment:
ruby style guide would recommend to reverse this if statement to have something like this:

```ruby
if my_element
  my_element.click
else
  other_logic
end
```

Reply:
And ```my_element``` is not good enough name for variable